scripts/netctl-tray.policy

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is the purpose of this file?

Reply:
This is for Polkit version <0.106. >=0.106 allows use of the javascript .rules files, but 0.105 and earlier don't. In order to allow the pkexec calls, I needed to make them explicit actions. And then rather than add them as both actions with default deny and a .pkla polkit localauthority file to allow them, I just added them as actions with default allow. Which is fine, because they're minimal scripts, without real attack surface. One lets you call netctl list, and the other lets you call switch-to but only to a profile that already exists.

I guess an attacker who had local code execution could get you to disconnect from your current profile, but they could already do that without local code execution by de-authing your wifi. I think the risk is pretty low.

Reply:
current Ubuntu ships polkit 0.105

Reply:
so this file is for compatibility with older polkit versions? Was it generated? If so, how did you generate it? Its pretty unreadable

Reply:
I wrote it based on the example for pkexec. All the action files in that directory look like this.